static/css/index-mobile.css

```diff
@@ -1,7 +1,6 @@
 #content
 {
   text-align: center;
-  padding-top: 2.5em;
```

Comment:
I'm still seeing the navbar overlap the first heading e.g. http://localhost:5000/en/about. The body content should be in the same place as it is currently e.g. https://adblockplus.org/en/about.

Reply:
The overlap is gone. But the spacing above the main content is still not the same as it was before this change. Please try to retain the spacing (or ask for help if it's not trivial / you're not up for it).